mathesar/urls.py

```diff
@@ -1,27 +1,34 @@
 from django.urls import include, path
 from rest_framework_nested import routers
 
-from mathesar.views import api, frontend
+from mathesar import views
+from mathesar.api.viewsets.columns import ColumnViewSet
+from mathesar.api.viewsets.constraints import ConstraintViewSet
+from mathesar.api.viewsets.data_files import DataFileViewSet
+from mathesar.api.viewsets.databases import DatabaseViewSet
+from mathesar.api.viewsets.records import RecordViewSet
+from mathesar.api.viewsets.schemas import SchemaViewSet
+from mathesar.api.viewsets.tables import TableViewSet
```

Comment:
It might be worth adding these to the `api` namespace for easier importing using the appropriate `__init__.py`.

Reply:
Good idea.